### DISCUSSION TITLE
[M-Z] (12 Settembre 2018) Esercizio 4

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#431] [Esami & Esoneri M-Z]
### Traccia

<img width="984" alt="image" src="https://github.com/user-attachments/assets/e7f97058-7f0a-44f5-885b-e157ad0a456e">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20180912.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20180912.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/b1abdbe8-13d8-45cd-ace4-d82baadd114b)
